tests/Shared/TempDirectory.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+public sealed class TempDirectory : IDisposable
+{
+    public string Path { get; } = Directory.CreateTempSubdirectory(".aspire-tests").FullName;
+
+    public void Dispose()
```

Comment:
Q: Is there a way to ensure users are at least attempting to dispose it (analyzer?). I assume it's not worth implementing a Finalizer for missed cases (only tests, no risk to no call using)?

Reply:
The only risk is an abandoned directory in TEMP.

The analyzer does catch it, but just enabling the analyzer on `tests` results in hundreds of warnings.

I tried a finalizer, but just running a few tests doesn't kick the GC in. I can leave the finalizer in just in case it catches something, if you want.

Reply:
All is good, thanks.